cmd/client/workflow.go

```diff
@@ -89,12 +78,13 @@ var WorkflowCreate = &cobra.Command{
 			exitOnError(err)
 		}
 
-		workflow, err := loadWorklow(workflowPath)
+		yml, err := loadWorklow(workflowPath)
 		if err != nil {
 			exitOnError(err)
 		}
 
-		if err := client.Create("workflow", &workflow); err != nil {
+		var workflow types.Workflow
+		if err := client.Create("workflow", string(yml), common.YAML, &workflow); err != nil {
```

Comment:
I would probably be more consistent to pass the workflow as an object, and not as a string